labs/authentication-passport/README.md

```diff
@@ -31,13 +31,15 @@ the one that LoopBack 4 authentication system wants.
 
 ### Simple Usage
 
-1. Create an instance of the passport strategy
+- 1. Create an instance of the passport strategy
```

Comment:
i think the original numbering looks good. wondering why we need to change it. :)

Reply:
@dhmlau If you look at the npm registry page: https://www.npmjs.com/package/@loopback/authentication-passport

They all show as `1.` instead of a series of numbers 😞

Reply:
_I think_ if you make the code snippet indented one more level, the number should be fine.